boltdesignsystem/bolt
# DS-737 multi-column form layout and grid spacing fix

## Jira

https://pegadigitalit.atlassian.net/browse/DS-737

## Summary

Fixed grid spacing issues and added a multi-column form layout demo.

## Details

1. Fixes a bug where grid items are overflowing the grid container
2. Removed CSS that's no longer relevant
3. Adds a demo page showcasing multi-column form layout using the grid component

## How to test

Run the branch locally and check the Grid docs and the "Demo: Multi-column Form" under the Form docs.
